src/main/java/core/basesyntax/service/impl/FruitListImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.FruitList;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitListImpl implements FruitList {
+    @Override
+    public List<FruitTransaction> getFruitList(List<FruitTransaction> list) {
```

Comment:
The method name `getFruitList` is not clear about what it does. It seems to be converting a list of `FruitTransaction` objects into a list with unique fruit names with default quantity. Consider renaming it to something more descriptive, like `createUniqueFruitTransactions`.